### DISCUSSION TITLE
Authentication - Policy UI improvements & Tester/Playground

### DISCUSSION THREAD

**Discussion by atnegrete** [supabase/supabase#1217] [Feature Requests]
- Similar to Firebase's rule playground. https://firebase.google.com/docs/rules/simulator

- "intelisense" on the policy editor 
![image](https://user-images.githubusercontent.com/22260151/115443168-89182180-a1d8-11eb-900f-bedd6b045c78.png)

**Comment by inian** [comment]
Hi @atnegrete, we are working on some improvements here starting with the storage policy editor. Stay tuned :)
